<launch>
	<arg name="isTest" default="false" />
	<arg name="stageArgs" value="$(find agv_exploration_test)/worlds/star/star.world" />
	<include file="$(find agv_exploration)/launch/default.launch" pass_all_args="true" />
</launch>
